packages/spor-datepicker-react/src/TimePicker.tsx

```diff
@@ -144,6 +144,8 @@ export const TimePicker = ({
         title={backwardsLabel}
         icon={<DropdownLeftFill24Icon />}
         onClick={handleBackwardsClick}
+        isDisabled={isDisabled}
+        style={isDisabled ? { backgroundColor: "transparent" } : {}}
```

Comment:
Disablede ghost-knapper har en grå bakgrunn for å vise at de er disabled. Det vil vi veldig spesifikt overstyre i dette tilfellet.

Det fungerte ikke å gjøre det med `backgroundColor="transparent"`, fordi det blir overstyrt av en selector i ChakraUI.